File: 프로그래머스/0/120868. 삼각형의 완성조건 （2）/삼각형의 완성조건 （2）.cs
using System;

public class Solution {
    public int solution(int[] sides) {
        int i = 1;
        int max = Math.Max(sides[0], sides[1]);
        int min = Math.Min(sides[0], sides[1]);
        int answer = (max + min - 1) - (max - min);
        
        return answer;
    }
}

File: 프로그래머스/0/120868. 삼각형의 완성조건 （2）/삼각형의 완성조건 （2）.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;

int solution(vector<int> sides) {
    int answer = 0;
    int max = std::max(sides[0], sides[1]);
    int min = std::min(sides[0], sides[1]);
    answer = 2 * min - 1;
    return answer;
}